packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -475,13 +481,14 @@ export async function getDevMiniflareOptions(
 					);
 				}
 
+				logUnknownTails(
+					workerOptions.tails,
+					userWorkers,
+					viteDevServer.config.logger.warn
+				);
+
 				return {
 					...workerOptions,
-					tails: filterTails(
-						workerOptions.tails,
-						userWorkers,
-						viteDevServer.config.logger.warn
-					),
```

Comment:
@penalosa Can you confirm that removing this won't reintroduce the crashes you fixed in https://github.com/cloudflare/workers-sdk/pull/9115?

Reply:
I think this should be fine, since as far as I know Miniflare will no longer fail to start with an unknown service, with the dev registry work. I'll quickly check